evap/evaluation/models.py

```diff
@@ -419,6 +419,7 @@ def publish(self):
             self.textanswer_set.delete()
         else:
             self.textanswer_set.filter(state=TextAnswer.HIDDEN).delete()
+            self.textanswer_set.update(original_answer=None)
```

Comment:
what happens if the `textanswer_set` contains a HIDDEN answer? does the update call succeed?

Reply:
i have a course, make one textanswer hidden, change another, publish the course. the hidden textanswer was deleted and the original answer of the other does not exist anymore.
that's what you asked, right? it works.

Reply:
oh, i thought textanswer_set was some property that would contain deleted textanswers after the first line and then cause problems in the second line, that was plain wrong.